src/Form.php

```diff
@@ -120,6 +120,7 @@ public function style($style = null)
     {
         if ($style) {
             $this->presenter->setStyle($this->config->get('styles.' . $style));
+            $this->presenter->setFormStyle($style);
```

Comment:
Preveri še metodo present() v class-u Laraplus\Form\Fields\Base\Element.

Tam se namreč tudi kliče setStyle in bo potrebno poklicati tudi setFormStyle (oz. setStyleName)